openshift-ironic.spec

```diff
@@ -0,0 +1,293 @@
+%global pyver 3
+%global full_release ironic-%{version}
+
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+Name:           openshift-ironic
+Epoch:          1
+Summary:        OpenShift Baremetal Hypervisor API (ironic)
+# (rpittau) we probably need to change this
+Version:        12.1.1
+Release:        1%{?dist}
+License:        ASL 2.0
+URL:            http://www.openstack.org
+# (rpittau) maybe indicate version here ?
+Source0:        ironic.tar.gz
+
+Source1:        openshift-ironic-api.service
+Source2:        openshift-ironic-conductor.service
+Source3:        ironic-rootwrap-sudoers
+Source4:        ironic-dist.conf
+Source5:        ironic.logrotate
+
+BuildArch:      noarch
+BuildRequires:  openstack-macros
+BuildRequires:  python%{pyver}-setuptools
+BuildRequires:  python%{pyver}-devel
+BuildRequires:  python%{pyver}-pbr
+BuildRequires:  openssl-devel
+BuildRequires:  libxml2-devel
+BuildRequires:  libxslt-devel
+BuildRequires:  gmp-devel
+BuildRequires:  python%{pyver}-sphinx
+BuildRequires:  systemd
+# Required to compile translation files
+BuildRequires:  python%{pyver}-babel
+# Required to run unit tests
+BuildRequires:  python%{pyver}-alembic
+BuildRequires:  python%{pyver}-automaton
+BuildRequires:  python%{pyver}-cinderclient
+BuildRequires:  python%{pyver}-eventlet
+BuildRequires:  python%{pyver}-futurist
+BuildRequires:  python%{pyver}-glanceclient
+BuildRequires:  python%{pyver}-jinja2
+BuildRequires:  python%{pyver}-jsonpatch
+BuildRequires:  python%{pyver}-jsonschema
+BuildRequires:  python%{pyver}-keystoneauth1
+BuildRequires:  python%{pyver}-keystonemiddleware
+BuildRequires:  python%{pyver}-mock
+BuildRequires:  python%{pyver}-neutronclient
+BuildRequires:  python%{pyver}-openstacksdk
+BuildRequires:  python%{pyver}-oslo-concurrency
+BuildRequires:  python%{pyver}-oslo-config
+BuildRequires:  python%{pyver}-oslo-context
+BuildRequires:  python%{pyver}-oslo-db
+BuildRequires:  python%{pyver}-oslo-db-tests
+BuildRequires:  python%{pyver}-oslo-i18n
+BuildRequires:  python%{pyver}-oslo-log
+BuildRequires:  python%{pyver}-oslo-messaging
```

Comment:
Similar

Reply:
same here for the check part, I have to test it though